Wrappers/Python/ccpi/processors/RingRemoval.py

```diff
@@ -0,0 +1,145 @@
+# -*- coding: utf-8 -*-
+
+from scipy.fftpack import fftshift, ifftshift, fft, ifft
+import numpy as np
+import pywt
```

Comment:
Requires `pywt` as dependency for CIL in [`meta.yaml`](https://github.com/vais-ral/CCPi-Framework/blob/master/recipe/meta.yaml#L38)

Reply:
added